go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/speakeasy-api/speakeasy
 
-go 1.22
```

Comment:
Don't know if we wanted to bump this yet

Reply:
yea I wasn't sure. I didn't commit this part at first but ran into this error: https://github.com/speakeasy-api/speakeasy/actions/runs/9881822029/job/27293402152#step:10:205